mod/htmlawed/classes/HtmlawedPlugin/ValidateInputHook.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace HtmlawedPlugin;
```

Comment:
We will need to decide on namespacing standards for plugins.

My thought is `Vendor\Plugin\Class`, so in this case `Elgg\Htmlawed\ValidateInputHook`